### DISCUSSION TITLE
[Question][Hw3] Make does not work

### DISCUSSION THREAD

**Discussion by john-mai-2605** [prosyslab-classroom/cs348-information-security#539] [Q&A]
I followed the environment setup in HW0 and execute this
eval $(opam env)

Then, when I execute make in the root, this error is reported:
can't open display
Makefile:8: recipe for target 'all' failed
make: *** [all] Error 11

I did this in the KCLOUD machine, with VS Code.

Can anyone help me pass this error

**Comment by RiceBiscuits** [comment]
Do you use XShell?

**Comment by john-mai-2605** [comment]
No I didnt. Should I use it?

**Comment by RiceBiscuits** [comment]
Nope. Is that whole error message? Could you upload whole error message?

**Comment by john-mai-2605** [comment]
![image](https://user-images.githubusercontent.com/50077665/116885180-8ef31580-ac62-11eb-81a8-5b821a9fdbed.png)
It is here

**Comment by mrpositron** [comment]
Why make asks for sudo priveleges? Hmm. I did it without sudo.

**Comment by RiceBiscuits** [comment]
I think you should find what is Error code 11. 
https://www.thegeekstuff.com/2010/10/linux-error-codes/

**Comment by john-mai-2605** [comment]
I re-clone the repo and somehow it works now. Thanks, everyone. I'll close the issue
